test/Libraries/DynamoPythonTests/PythonEvalTests.cs

```diff
@@ -171,6 +172,59 @@ public void CPythonEngineIncludesTraceBack()
             }
         }
 
+        [Test]
+        public void CPythonEngineWithErrorRaisesCorrectEvent()
+        {
+
+            var code = @"
+1
+";          count = 0;
```

Comment:
The code should be a single line for something as simple as this. Also, please don't do multiple sentences in a single line.

Reply:
can you clarify this comment?
`Also, please don't do multiple sentences in a single line.`

Reply:
You are closing the string and initializing `count` in the same line.